util/git/client.go

```diff
@@ -41,14 +41,15 @@ type Client interface {
 	Checkout(revision string) error
 	LsRemote(revision string) (string, error)
 	LsFiles(path string) ([]string, error)
+	LsLargeFiles() ([]string, error)
```

Comment:
There is one more struct which supposed to implement git client interface: https://github.com/argoproj/argo-cd/blob/master/reposerver/metrics/gitwrapper.go#L1-L1

Reply:
Yes, one little change, so much butterfly effect. Should have compiled all components before pushing that commit :|